cellmlmanip/units.py

```diff
@@ -243,6 +243,15 @@ def get_conversion_factor(self, quantity, to_unit):
         """Returns the magnitude multiplier required to convert from_unit to to_unit """
         return self.convert_to(quantity, to_unit).magnitude
 
+    def dimensionally_equivalent(self, symbol1, symbol2):
+        """Returns whether symbol1 and symbol2 are dimensionally_equivalent (same units ignoring a calling factor)"""
+        try:
+            self.get_conversion_factor(1 * self.summarise_units(symbol1),
```

Comment:
Why `1 * ` ???

Reply:
because `def get_conversion_factor(self, quantity, to_unit):`
`self.summarise_units(symbol1)`  is a unit and `1 * self.summarise_units(symbol1)` is a quanity

Reply:
Thanks!